### PR TITLE
	Fixes #9176: display incremental update task in nutupane, bz 1188823.

### PR DESCRIPTION
**Pull request by waldenraines**
http://projects.theforeman.org/issues/9176
https://bugzilla.redhat.com/show_bug.cgi?id=1188823

### CODE REVIEW

**Comment by waldenraines** [comment]
[test]

**Comment by ehelms** [comment]
[test]

**Comment by jlsherrill** [comment]
The 'back' button on the task details page isn't really useful here.  Maybe someway to hide it?

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/details/views/errata-details.html [hunk 1]
```diff
@@ -41,6 +41,12 @@ <h2 class="fl">{{ errata.title }}</h2>
           </span>
         </a>
       </li>
+      <li ng-class="{active: stateIncludes('errata.tasks')}">
+        <a translate
+           ui-sref="errata.tasks.index({errataId: errata.id})">
+          Tasks
+        </a>
```
Comment (jlsherrill) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/details/views/errata-details.html]: I don't think this tab is ever going to actually show anything as the tasks aren't tying themselves to the errata.  We might can do that, but its not currently being done.

**Comment by jlsherrill** [comment]
Also the tasks list for under errata details is missing the tabs 'details', 'content hosts', 'repositories', 'tasks'

**Comment by waldenraines** [comment]
@jlsherrill updated.

**Comment by ehelms** [comment]
Be nice to have some context of where I am in the details pane when ooking at the task -- and a way to navigate back to other parts of the errata details. Also a way to close the task details.

**Comment by waldenraines** [comment]
> Be nice to have some context of where I am in the details pane when ooking at the task -- and a way to navigate back to other parts of the errata details. Also a way to close the task details.

You aren't necessarily within the errata details, because you didn't necessarily select a single errata (i.e. this is the same state you arrive at after doing a bulk errata update).  In order to differentiate between the two I would need two states.  I can do this but how would we solve your context issue in the bulk case?

**Comment by jlsherrill** [comment]
Maybe in the bulk case back would just take you back to the main list?

**Comment by waldenraines** [comment]
> Maybe in the bulk case back would just take you back to the main list?

That's what I'm doing, in both cases.

**Comment by ehelms** [comment]
In the bulk case, I'd expect to see something at the top informing me I was applying a bulk errata apply of some sort with a close button that takes me back to the main list. In the single errata case, I'd expect task details within the context of the errata details pane since that is where I kicked it off from. I get the re-use pattern, but experience wise for me it just felt disconnected when I reviewed it.

**Comment by waldenraines** [comment]
@ehelms, @jlsherrill updated.

**Comment by ehelms** [comment]
I get:

```
Error: No such state 'errata.details.tasks.details'
    at Scope.transitionTo (angular-ui-router.js?body=1:1973)
    at success (apply-errata.controller.js?body=1:64)
    at angular-resource.js?body=1:555
    at deferred.promise.then.wrappedCallback (angular.js?body=1:10950)
    at angular.js?body=1:11036
    at Scope.$get.Scope.$eval (angular.js?body=1:11956)
    at Scope.$get.Scope.$digest (angular.js?body=1:11782)
    at Scope.$get.Scope.$apply (angular.js?body=1:12062)
    at done (angular.js?body=1:7844)
    at completeRequest (angular.js?body=1:8027)
```

**Comment by waldenraines** [comment]
@ehelms updated, sorry, didn't push my latest changes.

**Comment by ehelms** [comment]
ACK for me

**Comment by jlsherrill** [comment]
APJ

**Comment by waldenraines** [comment]
[test]

**Pull request merged**